src/tracer.ts

```diff
@@ -42,28 +82,10 @@ export default class Tracer {
      *     });
      *
      * @param {string} name - the name of the operation (REQUIRED).
-     * @param {object} [fields] - the fields to set on the newly created span.
-     * @param {string} [fields.operationName] - DEPRECATED: the name to use for
-     *        the newly created span. If provided, overrides the first argument
-     *        to startSpan(). Provided for backwards-compatibility.
-     * @param {SpanContext} [fields.childOf] - a parent SpanContext (or Span,
-     *        for convenience) that the newly-started span will be the child of
-     *        (per REFERENCE_CHILD_OF). If specified, `fields.references` must
-     *        be unspecified.
-     * @param {array} [fields.references] - an array of Reference instances,
-     *        each pointing to a causal parent SpanContext. If specified,
-     *        `fields.childOf` must be unspecified.
-     * @param {object} [fields.tags] - set of key-value pairs which will be set
-     *        as tags on the newly created Span. Ownership of the object is
-     *        passed to the created span for efficiency reasons (the caller
-     *        should not modify this object after calling startSpan).
-     * @param {number} [fields.startTime] - a manually specified start time for
-     *        the created Span object. The time should be specified in
-     *        milliseconds as Unix timestamp. Decimal value are supported
-     *        to represent time values with sub-millisecond accuracy.
+     * @param {SpanFields} [fields] - the fields to set on the newly created span.
      * @return {Span} - a new Span object.
      */
-    startSpan(name, fields) {
+    startSpan(name: string, fields: SpanFields = {}): Span {
```

Comment:
I'd prefer `s/fields/options/`, it seems to be a common pattern in JS libs and it is more accurate than "fields". Similarly, `s/SpanFields/StartSpanOptions/`